vignettes/handleiding_trendanalyse.Rmd

```diff
@@ -400,30 +455,170 @@ trendanalyse uit te kunnen voeren. Bij het uitvoeren van de trendanalyse
 in stap 4 moet er specifiek aandacht zijn voor de invloed van 
 rapportagegrenzen. Dit wordt verder toegelicht in stap 4. De 
 informatie in tabel 3 moet worden gebruikt bij de expert beoordeling
-van reeksen van waarnemingen in stap 4c.
+van reeksen van waarnemingen in stap 4d.
+
+#### *Toelichting goedkeuren en afkeuren van reeksen*
+In stap 3a, 3b en 3c worden reeksen afgekeurd op basis van het niet halen van de 
+criteria. 
+
+Het goedkeuren en afkeuren van reeksen gebeurt op basis van arbitraire 
+criteria. Met deze arbitraire criteria wordt onderscheid gemaakt in (1) 
+goedgekeurde reeksen waarvoor mogelijk trends kunnen worden benoemd op basis van 
+het aantal metingen, de verdeling in de tijd en het aantal metingen boven de 
+rapportagegrens en (2) reeksen waarvoor waarschijnlijk het aantal metingen niet 
+voldoende zijn om trends te kunnen bepalen of er te veel rapportagegrenzen 
+aanwezig zijn in een reeks.
 
-  d) *Selecteer de reeksen met >= 4 waarnemingen* Voor het uitvoeren
+In tabel 4 wordt een overzicht gegeven van het aantal reeksen en het aantal 
+goed- en afgekeurde reeksen. Deze informatie dient in stap 4 te worden gebruikt 
+bij de interpretatie van de trendresultaten en in stap 6 bij het komen tot een 
+uitspraak op niveau van een grondwaterlichaam. 
+
+  e) *overzicht gekeurde reeksen* Voor het uitvoeren
   van de statistische toets dienen ten minste 5 metingen in een reeks
   aanwezig te zijn. Dit betekent dat de trendanalyse alleen kan worden
   uitgevoerd als er ten minste 4 waarnemingen (<RG) en 1 rapportagegrens
   in een reeks zitten. Dit is een hard criterium. In deze stap worden
   de reeksen geselecteerd die aan dit criterium voldoen. Deze reeksen
   zijn goedgekeurd voor het uitvoeren van de statische toets (zie tabel 4).
   De overige reeksen worden niet meegenomen. 
+    
+  Een overzicht per put per stof per diepte kan verkregen worden met onderstaand 
+  voorbeeld. Reeksen met onvoldoende waarnemingen > rapportagegrens kunnen
+  vervolgens worden afgekeurd. Zie onderstaand script voor een voorbeeld (in het
+  voorbeeld staat meetverdeling voor meetfrequentie):
 
+```{r}
```

Comment:
Is dit codeblok op te knippen in kleinere stukken? Dan kan je in de tekst uitleggen wat je aan het doen bent. De 4-voudig geneste for loop is zowel pragmatisch als dramatisch.  Het zou helpen als je ipv h,i,j,k zinvolle variabele-namen zou gebruiken.